### PR TITLE
Address Pilot Gaps: Course-run Detail Page

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-6798](https://openedx.atlassian.net/browse/ECOM-6798)

### CODE REVIEW

**Comment by tasawernawaz** [comment]
@bderusha Please review it.

**Review by bderusha — APPROVED** [review]

File: course_discovery/apps/publisher/views.py [hunk 1]
```diff
@@ -160,6 +161,12 @@ def get_context_data(self, **kwargs):
                 )
             }
         ]
+
+        if mixins.check_roles_access(self.request.user):
```
Comment (waheedahmed) [inline course_discovery/apps/publisher/views.py]: Instead doing this you can simply assign like this:
```
context['can_view_tab'] = mixins.check_roles_access(self.request.user)
```

File: course_discovery/apps/publisher/views.py [hunk 2]
```diff
@@ -160,6 +161,12 @@ def get_context_data(self, **kwargs):
...
+            context['can_view_tab'] = True
```
Comment (waheedahmed) [inline course_discovery/apps/publisher/views.py]: ```is_internal_user```

File: course_discovery/templates/publisher/course_run_detail.html [hunk 3]
```diff
@@ -58,11 +63,18 @@ <h2 class="hd-2 emphasized">
                 <div id="tab-5" class="tab-content">
                     {% include 'publisher/course_run_detail/_salesforce.html' %}
                 </div>
+              {% else %}
+                <div id="tab-2" class="tab-content active">
+                    {% include 'publisher/course_run_detail/_all.html' %}
```
Comment (waheedahmed) [inline course_discovery/templates/publisher/course_run_detail.html]: Move this outside else above if, no need to add else part.

**Review by waheedahmed — CHANGES_REQUESTED** [review]

File: course_discovery/apps/publisher/views.py (hunk 2 above)
Comment (tasawernawaz) [reply course_discovery/apps/publisher/views.py]: renamed it to 'can_view_all_tabs' like @awais786  is doing in #549

**Comment by tasawernawaz** [comment]
@waheedahmed Feedback addressed you can review again.

**Review by waheedahmed — APPROVED** [review]

**Pull request merged**